### DISCUSSION TITLE
How to retrieve user email/name from `user_id` in JSON which downloaded from `Export All Chats (All Users)`?

### DISCUSSION THREAD

**Discussion by changchiyou** [open-webui/open-webui#1030] [Q&A] [answered]
I attempted to retrieve this information using `<open-webui-server>/api/v1/users`:

```
[
  {
    "id": "string",
    "name": "string",
    "email": "string",
    "role": "pending",
    "profile_image_url": "/user.png",
    "timestamp": 0
  }
]
```

However, it seems that the `id` in this response is not equivalent to the `user_id`.

**Comment by changchiyou** [comment, marked as answer]
Never mind, I accidentally deleted the volume, so I recreated the account, resulting in a new account ID that differs from the old one in the previously exported JSON. 😭

**Comment by ThomasKubus** [comment]
Hi, im struggling with the API Call to get the users, how i exactly does the API call look? I have only managed to call /api/chat so far

**Reply by changchiyou** [reply]
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/01898582-c446-4796-943a-b44d2c5c931a" />

**Reply by ThomasKubus** [reply]
Thanks for the quick answer! I still get 404 page not found though :( I'll keep on searching
